src/Presentation/Nop.Web/App_Data/Localization/defaultResources.nopres.xml

```diff
@@ -16533,6 +16539,12 @@
   <LocaleResource Name="Checkout.UseRewardPoints">
     <Value>Use my reward points, {0} of {1} reward points ({2}) available for this order</Value>
   </LocaleResource>
+  <LocaleResource Name="Checkout.VatNumber">
+    <Value>VAT number</Value>
+  </LocaleResource>
+  <LocaleResource Name="Checkout.VatNumber.Warning">
+    <Value>VAT number can be entered and used only after registration.</Value>
```

Comment:
We could add a link to the customer info page here so that the user can go, register and immediately enter the VAT number.